tgui/packages/tgui/interfaces/CrewConsole.js

```diff
@@ -164,7 +192,7 @@ const CrewTableEntry = (props, context) => {
         )}
       </Table.Cell>
       <Table.Cell>
-        {area !== undefined ? area : 'N/A'}
+        {area !== undefined ? area : <Icon name="question" color="#ffffff" size={1} /> }
```

Comment:
skyrat comment

Reply:
ah yeah that was a last minute one i overlooked